Generate.py

```diff
@@ -204,15 +204,16 @@ def main(args=None) -> tuple[argparse.Namespace, int]:
     player_path_cache = {}
     for player in range(1, args.multi + 1):
         player_path_cache[player] = player_files.get(player, args.weights_file_path)
-    name_counter = Counter()
+    name_counter: Counter = Counter()
```

Comment:
What does this do? Is there a type checker that doesn't know that `name_counter` is a `Counter` if we just say `name_counter = Counter()`?

Reply:
(I think it would be better if the annotation were `Counter[str]`, because that's information you can't see just from looking at `Counter()`)

Reply:
mypy gives an error without it similarly to other container types, although pyright doesn't care here. I'd be fine with either removing it or adding the type parameter, although for the latter I think it should also be added to the annotation for the parameter in `handle_name`.

Reply:
Yeah, I think `Counter[str]` is a better way to go (along with the parameter in `handle_name`).

I would recommend using `--disallow-any-generics` with mypy, so that it's not leading to these annotations that don't add any useful information.